### PR TITLE
fixing dependency on jersey

### PR DESCRIPTION
**Pull request by shroffk**

### CODE REVIEW

File: applications/channel/channel-plugins/org.csstudio.utility.channelfinder/src/org/csstudio/utility/channelfinder/Activator.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
      * The constructor
      */
     public Activator() {
+    	System.out.println("Starting 1");
```
Comment (willrogers) [inline applications/channel/channel-plugins/org.csstudio.utility.channelfinder/src/org/csstudio/utility/channelfinder/Activator.java]: Maybe you didn't want to include this line.

**Comment by berryma4** [comment]
@shroffk reference issue number?

**Comment by shroffk** [comment]
Issue #1417

File: applications/channel/channel-plugins/org.csstudio.utility.channelfinder/src/org/csstudio/utility/channelfinder/Activator.java (hunk 1 above)
Comment (shroffk) [inline applications/channel/channel-plugins/org.csstudio.utility.channelfinder/src/org/csstudio/utility/channelfinder/Activator.java]: Thank you, I have cleaned this.

**Pull request merged**